test/index.ts

```diff
@@ -26,7 +26,7 @@ const example = require(`../src/strategies/${strategy}/examples.json`)[0];
   console.time('getScores');
   try {
     const scores = await snapshot.utils.getScores(
-      'yam.eth',
+      'frax-finance',
```

Comment:
Can you remove the changes on this file?

Reply:
done